lib/services/slack.rb

```diff
@@ -15,21 +15,31 @@ class Service::Slack < Service::Base
   page 'Username', [:username]
 
   def receive_verification(config, _)
-    send_message(config, receive_verification_message)
-    [true, "Successfully sent a message to channel #{ config[:channel] }"]
-  rescue => e
-    log "Rescued a verification error in Slack: #{ e.message }"
-    [false, "Could not send a message to channel #{ config[:channel] }. #{e.message}"]
+    response = send_message(config, receive_verification_message)
+
+    if response.code == '200'
+      [true, "Successfully sent a message to channel #{ config[:channel] }"]
+    else
+      [false, error_response_message(response)]
+    end
   end
 
   def receive_issue_impact_change(config, payload)
     message, options = format_issue_impact_change_message(payload)
-    send_message(config, message, options)
-    :no_resource
+    response = send_message(config, message, options)
+    if response.code == '200'
+      :no_resource
```

Comment:
fixed, I cut over a bunch of these `:no_resource` to use true instead, and updated the base class documentation to reflect this change (and also explain the historic reason that some of the other service hooks still return hashy values)